php/class-fieldmanager-field.php

```diff
@@ -415,7 +482,10 @@ public function element_markup( $values = array() ) {
 
 		$out = '';
 
-		// If this element is part of tabbed output, there needs to be a wrapper to contain the tab content
+		/**
```

Comment:
Multi-line comments should use `/*` instead of `/**`, [per the docs standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-2-multi-line-comments) (applies throughout).